cinder/volume/drivers/nexenta/iscsi.py

```diff
@@ -113,13 +114,15 @@ def __init__(self, *args, **kwargs):
                               'option': option})
                 raise jsonrpc.NmsException(code='EINVAL', message=message)
         self.nms = None
+        self.san_root = None
         self.mappings = {}
         self.driver_name = self.__class__.__name__
         self.san_host = self.configuration.nexenta_host
         self.volume = self.configuration.nexenta_volume
         self.folder = self.configuration.nexenta_folder
         self.volume_compression = (
             self.configuration.nexenta_dataset_compression)
+        self.volume_dedup = self.configuration.nexenta_dataset_dedup
```

Comment:
Do we actually want to support dedup?

Reply:
Yes, by some our Customers. BTW: dedup is disabled by default.